server/model/monitor.js

```diff
@@ -1233,7 +1264,7 @@ class Monitor extends BeanModel {
      * @param {Monitor} monitor The monitor to send a notificaton about
      * @param {Bean} bean Status information about monitor
      */
-    static async sendNotification(isFirstBeat, monitor, bean) {
+    static async sendNotification(isFirstBeat, monitor, bean, downTime = 0) {
```

Comment:
```suggestion
    static async sendNotification(isFirstBeat, monitor, bean, downTime = null) {
```

Reply:
I am not sure I really understand the point of setting it to `null`

Reply:
I am not sure I really understand the point of setting it to null